HW04/ihar_shkuryn/spec/homework_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative '../lib/homework'
+require_relative '../lib/student'
+require_relative '../lib/mentor'
+require_relative 'spec_helper'
+
+RSpec.describe Homework do
+  let(:title) { 'HW03' }
+  let(:description) { 'description homework' }
+  let(:student) { Student.new(name: 'John', surname: 'Doe') }
+  let(:mentor) { Mentor.new(name: 'Jack', surname: 'Gonsales') }
+
+  let(:obj) { described_class.new(title, description, student, mentor, 'new') }
+
+  describe 'checking fields of instance' do
+    context 'when fields is correct' do
+      it 'returns instance of Homework' do
```

Comment:
I think is test is useless. You are basically testing ruby and it's classes instance creation possibilities